package-support/sysconfig/sideboard

```diff
@@ -0,0 +1,6 @@
+# defaults for sideboard package
```

Comment:
This is a refinement of the original way we did ``init.d`` scripts (again, MAGFest doesn't care, but I'll explain for anyone interested).

Sideboard can run in two modes:
- "server" mode is where we start CherryPy and serve up APIs and webpages to clients
- "daemon" mode is where we just run background tasks (e.g. ``DaemonTask`` stuff)

We want to have a single ``init.d`` script for both of these cases.  What we do us have the ``init.d`` script include this ``sysconfig`` file, which controls the config settings used by the ``init.d`` script.  This allows us to do things like have Puppet (or your provisioning tool of choice) update this file to determine what mode Sideboard runs, without needing Puppet to actually manage your ``init.d`` script.